### PR TITLE
adding messages to the channels

### PR DESCRIPTION
**Pull request by JuanDavila1101**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by jim-conner — APPROVED** [review]
Great job, especially with MessagesViewChoice!

File: src/App/components/SideBar.js [hunk 1]
```diff
@@ -25,12 +24,19 @@ function SideBar({
       deleteChannelUsers(channelID);
       deleteChannel(channelID).then(setChannels);
     };
+
+    const history = useHistory();
+
+    const goToConversationView = (firebaseKey) => {
+      history.push(`direct-messages/${firebaseKey}`);
```
Comment (Gabrielle-Tobermann) [inline src/App/components/SideBar.js:31]: Can you change that history.push to ```add-channel/${firebaseKey}```? When I click on a channel the url is incorrect (it says direct-messages)

Comment (Gabrielle-Tobermann) [reply src/App/components/SideBar.js:31]: Also, if you move that useHistory (line 28) to right under line 20, it'll get rid of that warning about the order of hooks.

Comment (JuanDavila1101) [reply src/App/components/SideBar.js:31]: got the line moved to get rid of the warning, Thank you ma'am

**Review by Gabrielle-Tobermann — APPROVED** [review]

**Pull request merged**